elyra/cli/pipeline_app.py

```diff
@@ -155,6 +160,8 @@ def _preprocess_pipeline(pipeline_path: str,
         primary_pipeline["app_data"]["runtime"] = runtime
     if runtime_config:
         primary_pipeline["app_data"]["runtime-config"] = runtime_config
+    else:
+        primary_pipeline["app_data"]["runtime-config"] = 'local'
```

Comment:
I still don't feel comfortable defaulting here... have you seen a case where we are not passing the proper 'local' for local runs?

Reply:
The goal was to prevent a fatal downstream error if no runtime config value is specified _and_ subsequently an attempt is made to run the pipeline. I reverted the change as requested.